pkg/plugins/manager/signature/manifest.go

```diff
@@ -57,30 +59,47 @@ func (m *PluginManifest) isV2() bool {
 	return strings.HasPrefix(m.ManifestVersion, "2.")
 }
 
+// ModuleHash returns the SRI hash for the module.js file, as expected by the browser.
+// It returns an empty string if there's no module.js in the manifest.
+func (m *PluginManifest) ModuleHash() (string, error) {
```

Comment:
Could there be nested module.js files? 🤔

Reply:
For nested plugins I mean

Reply:
Good point. Yes there can be nested module.js files in the manifest:

```
"module.js": "efd8db933b56abcfd33f8c4bd66b29b074bc8e696dab6d2efff683e0cae6ca3c",
"datasource/module.js": "d5d85ee941585c34a9a3f0ceb38e45ef16c3691aae3e1f056efa1db2ce3cb251",
"panel-triggers/module.js": "7ee059fb3b8f4d28e6f5468486e22d07ab34fd0d6ce79c168227f0ac8c7a0353",
```

Reply:
Great catch. Fixed with the latest iteration :smiley: